mocks/mocks.go

```diff
@@ -339,23 +339,53 @@ func (*Publisher) SubmitToCT(_ context.Context, der []byte) error {
 	return nil
 }
 
-// Statter is a stat counter that is a no-op except for locally handling Inc
-// calls (which are most of what we use).
-type Statter struct {
+// MockStatter is a stat counter that is a no-op except for locally handling
```

Comment:
Why did you rename this? I can't see any advantage and it reintroduces the package stutter.